Python/Tests/Core/WorkspaceInterpreterFactoryTests.cs

```diff
@@ -159,15 +159,74 @@ public class WorkspaceInterpreterFactoryTests {
             }
         }
 
-        private static IEnumerable<InterpreterConfiguration> TestTriggerDiscovery(IVsFolderWorkspaceService workspaceService, Action triggerDiscovery) {
+        [TestMethod, Priority(0)]
+        public void WatchWorkspaceVirtualEnvRenamed() {
+            const string ENV_NAME = "env";
+            var workspaceService = CreateEnvAndGetService(ENV_NAME);
+
+            string envPath = Path.Combine(workspaceService.CurrentWorkspace.Location, ENV_NAME);
+            var configs = TestTriggerDiscovery
+                (workspaceService, () => Directory.Move(envPath, string.Concat(envPath, "1"))).ToArray();
```

Comment:
This is concatenating a folder name--not the path. If the starting variable is "C:\env", I want the string to become "C:\env1", where env1 is the new file name.

Reply:
It might work in this situation because you get a folder path in a certain way where the path doesn't end in `\`. But that's a bad thing to assume. Folder paths, depending on where you get them from, may or may not end in `\`.  Ex: `C:\env` vs `C:\env\` are both valid ways of representing that folder.

Reply:
Ah, I see what you mean. Is this a better approach for obtaining the new path? 

string renamedEnvPath = Path.Combine(workspaceService.CurrentWorkspace.Location, string.Concat(ENV_NAME, "1"));

Reply:
Yeah, that will work.